style.css

```diff
@@ -765,3 +765,137 @@ button {
   position: absolute;
   left: 133px;
 }
+
```

Comment:
-  [OPTIONAL] I highly recommend avoiding two specific things when it comes to styling with CSS  👍🏼 
    - Avoid setting `position: absolute` since it messes up the normal flow of the elements (Browsers add elements in a stack ordered ✌🏼 
    - Since you are practicing the Mobile First design approach ensure that all section takes up the whole screen width instead of setting a fixed width for each section 👍🏼 

![image](https://user-images.githubusercontent.com/68030297/230653416-85857e7e-39d0-4634-9976-2eeba56641ae.png)

Reply:
sure I will do that!